lib/ui/screens/my_account/widgets/account_asset_info.dart

```diff
@@ -20,14 +21,13 @@ import '../../../commons/custom_alert_widget.dart';
 
 class AccountAssetInfo extends ConsumerStatefulWidget {
   final bool? shrinkwrap;
-  final bool isSelected;
   final int index;
+  final bool isForSelectAccount;
```

Comment:
why change the name?

Reply:
Because we implemented newly fresh code from the #969 branch, In the old code was **isSelected** instead of **isForSelectAccount**.

Reply:
i know ~ but i changed it ~ why changing it back?